src/findcdn/cdnEngine/detectCDN/cdn_check.py

```diff
@@ -59,7 +95,7 @@ def __init__(self):
 
     def ip(self, dom: Domain) -> List[int]:
         """Determine IP addresses the domain resolves to."""
-        dom_list: List[str] = [dom.url, "www." + dom.url]
```

Comment:
My main motivation was to add www. for domains which don't directly answer to their domain name but instead they will listen to www.domain.com. Is there a reason why you wanted to remove "www." ? 

I have also thought about pairing in a light subdomain enumerator using like google but I think that's too big of a scope.